rancid-tomatillos/src/App.js

```diff
@@ -22,15 +23,17 @@ class App extends Component {
     fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
       .then(response => {
         if (!response.ok) {
-          throw new Error(`Error: ${response.status}`)
+          throw new Error(`${response.status}`)
         } else {
           return response.json();
         }
       })
       .then(data => {
         this.setState({ movies: data.movies })
       })
-      .catch(err => console.log(err))
+      .catch(err => {
```

Comment:
I think this was a better way to handle error instead of console.logging, good job!